KarhooUISDK/Screens/CheckoutScreen/Components/Loyalty/LoyaltyView.swift

```diff
@@ -41,12 +39,33 @@ struct LoyaltyView: View {
     }
     
     @ViewBuilder
-    private var earnContent: some View {
-        let pointsEarnedText = String(
+    private var viewOverOrSeparator: some View {
+        VStack(alignment: .leading) {
+            loyaltyTitle
```

Comment:
Why add the title inside the earn container?

Reply:
Because I wanted to have one accessibility element with one accessibilityValue containing both texts